config/config.go

```diff
@@ -596,7 +598,9 @@ type Cache struct {
 	// on new request and re-cached
 	Expire Duration `yaml:"expire,omitempty"`
 
-	// Grace duration before the expired entry is deleted from the cache.
+	// Deprecated: GraceTime duration before the expired entry is deleted from the cache.
+	// It's deprecated and in future versions it'll be replaced by user's MaxExecutionTime.
+	// It's already the case today if value of GraceTime is omitted.
 	GraceTime Duration `yaml:"grace_time,omitempty"`
```

Comment:
don't forget to deprecate grace_time in the https://github.com/ContentSquare/chproxy/blob/master/config/README.md